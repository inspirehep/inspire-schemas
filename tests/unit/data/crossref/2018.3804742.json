{
   "status":"ok",
   "message-version":"1.0.0",
   "message":{
      "DOI":"10.1155/2018/3804742",
      "subtitle":[

      ],
      "reference":[
         {
            "key":"14",
            "year":"1999"
         },
         {
            "volume":"491",
            "series-title":"Lecture Notes in Economics and Mathematical Systems",
            "key":"11",
            "year":"2000"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.1007/bf01210689",
            "key":"9"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.1007/BF01197716",
            "key":"18"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.1016/j.ejor.2017.03.041",
            "key":"25"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.1016/j.mathsocsci.2014.12.004",
            "key":"8"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.2143/AST.38.1.2030405",
            "key":"4"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.1016/j.ejor.2017.04.029",
            "key":"3"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.1007/s001860200207",
            "key":"12"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.1016/j.ejor.2017.04.012",
            "key":"6"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.1016/S1574-0110(02)80014-5",
            "key":"10"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.1006/jeth.2001.2814",
            "key":"16"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.1016/j.jmateco.2011.07.003",
            "key":"13"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.1007/s11083-011-9230-4",
            "key":"15"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.1007/s11238-015-9506-z",
            "key":"2"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.1016/S0165-4896(01)00083-X",
            "key":"22"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.1007/s40505-016-0106-z",
            "key":"7"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.1016/j.ejor.2013.02.045",
            "key":"19"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.1016/0377-2217(80)90195-2",
            "key":"24"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.2307/1909773",
            "key":"21"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.2307/1909243",
            "key":"17"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.1090/S0002-9939-1954-0063016-5",
            "key":"23"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.1016/0022-0531(75)90009-5",
            "key":"5"
         },
         {
            "doi-asserted-by":"publisher",
            "DOI":"10.1007/PL00004219",
            "key":"1"
         }
      ],
      "issued":{
         "date-parts":[
            [
               2018
            ]
         ]
      },
      "abstract":"<jats:p>We characterize the existence of (weak) Pareto optimal solutions to the classical multiobjective optimization problem by referring to the naturally associated preorders and their finite (Richter-Peleg) multiutility representation. The case of a compact design space is appropriately considered by using results concerning the existence of maximal elements of preorders. The possibility of reformulating the multiobjective optimization problem for determining the weak Pareto optimal solutions by means of a scalarization procedure is finally characterized.</jats:p>",
      "short-title":[

      ],
      "prefix":"10.1155",
      "relation":{
         "cites":[

         ]
      },
      "subject":[
         "Applied Mathematics",
         "Analysis"
      ],
      "author":[
         {
            "affiliation":[
               {
                  "name":"DIA, Università di Trieste, 34127 Trieste, Italy"
               }
            ],
            "given":"Paolo",
            "family":"Bevilacqua"
         },
         {
            "ORCID":"http://orcid.org/0000-0002-3799-4630",
            "authenticated-orcid":"True",
            "given":"Gianni",
            "family":"Bosi",
            "affiliation":[
               {
                  "name":"DEAMS, Università di Trieste, 34127 Trieste, Italy"
               }
            ]
         },
         {
            "affiliation":[
               {
                  "name":"DEM, Università di Brescia, 25122 Brescia, Italy"
               }
            ],
            "given":"Magalì",
            "family":"Zuanon"
         }
      ],
      "reference-count":24,
      "ISSN":[
         "1085-3375",
         "1687-0409"
      ],
      "member":"98",
      "source":"Crossref",
      "score":1.0,
      "deposited":{
         "timestamp":1517182278000,
         "date-time":"2018-01-28T23:31:18Z",
         "date-parts":[
            [
               2018,
               1,
               28
            ]
         ]
      },
      "indexed":{
         "timestamp":1517184622360,
         "date-time":"2018-01-29T00:10:22Z",
         "date-parts":[
            [
               2018,
               1,
               29
            ]
         ]
      },
      "type":"journal-article",
      "URL":"http://dx.doi.org/10.1155/2018/3804742",
      "is-referenced-by-count":0,
      "volume":"2018",
      "issue": "1",
      "issn-type":[
         {
            "type":"print",
            "value":"1085-3375"
         },
         {
            "type":"electronic",
            "value":"1687-0409"
         }
      ],
      "link":[
         {
            "URL":"http://downloads.hindawi.com/journals/aaa/2018/3804742.pdf",
            "intended-application":"text-mining",
            "content-version":"vor",
            "content-type":"application/pdf"
         },
         {
            "URL":"http://downloads.hindawi.com/journals/aaa/2018/3804742.xml",
            "intended-application":"text-mining",
            "content-version":"vor",
            "content-type":"application/xml"
         },
         {
            "URL":"http://downloads.hindawi.com/journals/aaa/2018/3804742.pdf",
            "intended-application":"similarity-checking",
            "content-version":"vor",
            "content-type":"unspecified"
         }
      ],
      "published-print":{
         "date-parts":[
            [
               2018
            ]
         ]
      },
      "references-count":24,
      "original-title":[

      ],
      "short-container-title":[
         "Abstract and Applied Analysis"
      ],
      "publisher":"Hindawi Limited",
      "content-domain":{
         "domain":[

         ],
         "crossmark-restriction":"False"
      },
      "license":[
         {
            "URL":"http://creativecommons.org/licenses/by/4.0/",
            "start":{
               "timestamp":1514764800000,
               "date-time":"2018-01-01T00:00:00Z",
               "date-parts":[
                  [
                     2018,
                     1,
                     1
                  ]
               ]
            },
            "content-version":"unspecified",
            "delay-in-days":0
         }
      ],
      "created":{
         "timestamp":1517182278000,
         "date-time":"2018-01-28T23:31:18Z",
         "date-parts":[
            [
               2018,
               1,
               28
            ]
         ]
      },
      "title":[
         "Multiobjective Optimization, Scalarization, and Maximal Elements of Preorders"
      ],
      "article-number": "3804742",
      "alternative-id":[
         "3804742",
         "3804742"
      ],
      "container-title":[
         "Abstract and Applied Analysis"
      ],
      "page":"1-6"
   },
   "message-type":"work"
}
