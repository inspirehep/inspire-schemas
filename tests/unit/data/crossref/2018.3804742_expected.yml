---
license:
- url: http://creativecommons.org/licenses/by/4.0/
  imposing: Hindawi Limited
  material: publication
authors:
- raw_affiliations:
  - source: Crossref
    value: DIA, Università di Trieste, 34127 Trieste, Italy
  full_name: Bevilacqua, Paolo
- raw_affiliations:
  - source: Crossref
    value: DEAMS, Università di Trieste, 34127 Trieste, Italy
  ids:
  - value: 0000-0002-3799-4630
    schema: ORCID
  full_name: Bosi, Gianni
- raw_affiliations:
  - source: Crossref
    value: DEM, Università di Brescia, 25122 Brescia, Italy
  full_name: Zuanon, Magalì
journal_volume: '2018'
artid: '3804742'
journal_issue: '1'
page_start: '1'
page_end: '6'
year: 2018
material: publication
journal_title: Abstract and Applied Analysis
title: Multiobjective Optimization, Scalarization, and Maximal Elements of Preorders
references:
- reference:
    publication_info:
      year: 1999
- reference:
    publication_info:
      year: 2000
      journal_volume: '491'
- reference:
    dois:
    - 10.1007/bf01210689
- reference:
    dois:
    - 10.1007/BF01197716
- reference:
    dois:
    - 10.1016/j.ejor.2017.03.041
- reference:
    dois:
    - 10.1016/j.mathsocsci.2014.12.004
- reference:
    dois:
    - 10.2143/AST.38.1.2030405
- reference:
    dois:
    - 10.1016/j.ejor.2017.04.029
- reference:
    dois:
    - 10.1007/s001860200207
- reference:
    dois:
    - 10.1016/j.ejor.2017.04.012
- reference:
    dois:
    - 10.1016/S1574-0110(02)80014-5
- reference:
    dois:
    - 10.1006/jeth.2001.2814
- reference:
    dois:
    - 10.1016/j.jmateco.2011.07.003
- reference:
    dois:
    - 10.1007/s11083-011-9230-4
- reference:
    dois:
    - 10.1007/s11238-015-9506-z
- reference:
    dois:
    - 10.1016/S0165-4896(01)00083-X
- reference:
    dois:
    - 10.1007/s40505-016-0106-z
- reference:
    dois:
    - 10.1016/j.ejor.2013.02.045
- reference:
    dois:
    - 10.1016/0377-2217(80)90195-2
- reference:
    dois:
    - 10.2307/1909773
- reference:
    dois:
    - 10.2307/1909243
- reference:
    dois:
    - 10.1090/S0002-9939-1954-0063016-5
- reference:
    dois:
    - 10.1016/0022-0531(75)90009-5
- reference:
    dois:
    - 10.1007/PL00004219
dois:
- doi: 10.1155/2018/3804742
  material: publication
document_type: article
abstract: "<jats:p>We characterize the existence of (weak) Pareto optimal solutions to the classical multiobjective optimization problem by referring to the naturally associated preorders and their finite (Richter-Peleg) multiutility representation. The case of a compact design space is appropriately considered by using results concerning the existence of maximal elements of preorders. The possibility of reformulating the multiobjective optimization problem for determining the weak Pareto optimal solutions by means of a scalarization procedure is finally characterized.</jats:p>"
imprints: '2018'
